src/components/Topics.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+import { fetchTopics } from "../assets/axios";
+
+import "../styles/Topics.css"
+
+
+export default function Topics() {
+    const [topics, setTopics] = useState([])
+    const [isLoading, setLoading] = useState(true)
+
+
+    useEffect(() => {
+        fetchTopics()
```

Comment:
Really good that you're fetching the topics so that it will grab all of them from the database 👍